scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -16,260 +16,352 @@
-        case $yn in
-            [Yy]*) return 0  ;;
-            [Nn]*) return  1 ;;
-        esac
-    done
-}
-
-if [ -f /etc/lsb-release ]; then
-    # Ubuntu
-    echo
-    echo "This script installs the following OpenVINO 3rd-party dependencies:"
-    echo "  1. GTK+, FFmpeg and GStreamer libraries used by OpenCV"
-    echo "  2. libusb library required for Myriad plugin for Inference Engine"
-    echo "  3. build dependencies for OpenVINO samples"
-    echo "  4. build dependencies for GStreamer Plugins"
-    echo
-    PKGS=(
-        cpio
-        build-essential
-        cmake
-        libusb-1.0-0-dev
-        libdrm-dev
+pkgs=()
+extra_repos=()
+
+if [ "$os" == "ubuntu18.04" ] ; then
+
+    # OpenCV
+    pkgs+=(libgtk-3-0)
+    [ -n "$optional" ] && pkgs+=(
+        libavcodec57
+        libavformat57
+        libavutil55
+        libswscale4
+        libavresample3
         libgstreamer1.0-0
         gstreamer1.0-plugins-base
         gstreamer1.0-plugins-good
+        gstreamer1.0-plugins-ugly
         gstreamer1.0-plugins-bad
-        gstreamer1.0-vaapi
-        ffmpeg
     )
-    system_ver=$(cat /etc/lsb-release | grep -i "DISTRIB_RELEASE" | cut -d "=" -f2)
-    if [ "$system_ver" = "16.04" ]; then
-        PKGS+=( libgtk2.0-0 )
-    else
-        if [ "$system_ver" = "20.04" ]; then
-            PKGS+=( gstreamer1.0-plugins-ugly
-                    gstreamer1.0-libav
-                    libgstreamer-plugins-base1.0-dev
-                    gstreamer1.0-alsa
-                    libgstrtspserver-1.0-dev
-                    python3-gst-1.0
-                    libfluidsynth2
-                    libnettle7
-                    libopenexr24
-                    python3.8
-                    libpython3.8
-                    libglib2.0-0
-                )
-        elif [ "$system_ver" = "18.04" ]; then
-            PKGS+=( libfluidsynth1
-                    libnettle6
-                    libopenexr22
-                    gstreamer1.0-plugins-ugly
-                    gstreamer1.0-alsa
-                    libglib2.0
-                )
-        fi
-        PKGS+=( flex
-                libgl-dev
-                libtag-extras1
-                libusb-1.0-0-dev
-                libfaac0
-                python3-gi
-                libgtk-3-0
-        )
-    fi
-    apt update
-    # shellcheck disable=SC2068
-    apt install -y ${PKGS[@]}
-else
-    # CentOS
-    echo
-    echo "This script installs the following OpenVINO 3rd-party dependencies:"
-    echo "  1. GTK+ and GStreamer libraries used by OpenCV"
-    echo "  2. libusb library required for Myriad plugin for Inference Engine"
-    echo "  3. Python 3.6 for Model Optimizer"
-    echo "  4. gcc 4.8.5 and other build dependencies for OpenVINO samples"
-    echo "  5. build dependencies for GStreamer Plugins"
-    echo
-    PKGS=(
-        libusbx-devel
-        gtk2
+
+    # Others
+    # TODO: cleanup and choose optional
+    pkgs+=(
+        # cpio
```

Comment:
AFAIK, cpio was required by installer itself.

Reply:
Addressed in new version